### PR TITLE
Add account_cash_discount_reconcile_write_off

### PR DESCRIPTION
**Pull request by FranzPoize**
Adds a line to the info popover to display dicount amount and discount_due_date for account.move.line in the reconcile widget account_reconcile_write_off

### CODE REVIEW

File: account_cash_discount_reconcile_write_off/README.rst [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+Do not contact contributors directly about support or help with technical issues.
+
+Credits
+=======
+
+Authors
+~~~~~~~
+
+* ACSONE SA/NV
```
Comment (hparfr) [inline account_cash_discount_reconcile_write_off/README.rst]: please fix !

File: account_cash_discount_reconcile_write_off/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Copyright 2018 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Account Cash Discount Reconciliation Write off display",
+    "version": "14.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "ACSONE SA/NV," "Odoo Community Association (OCA)",
```
Comment (hparfr) [inline account_cash_discount_reconcile_write_off/__manifest__.py]: here too

File: account_cash_discount_reconcile_write_off/readme/USAGE.rst [hunk 3]
```diff
@@ -0,0 +1,6 @@
+To see the discount amount:
+
+#. Click on reconcile items for a bank account
+#. Add a partner that has a invoice with a cash discount to a payment
+#. Adds a invoice that has a cash discount
```
Comment (hparfr) [inline account_cash_discount_reconcile_write_off/readme/USAGE.rst]: typo

**Review by hparfr — CHANGES_REQUESTED** [review]
please squash in one commit

File: account_cash_discount_reconcile_write_off/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,17 @@
+# Copyright 2018 ACSONE SA/NV
```
Comment (nayatec) [inline account_cash_discount_reconcile_write_off/__manifest__.py]: pls fix

File: account_cash_discount_reconcile_write_off/README.rst [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+.. contents::
+   :local:
+
+Usage
+=====
+
+To see the discount amount:
+
+#. Click on reconcile items for a bank account
+#. Add a partner that has a invoice with a cash discount to a payment
```
Comment (nayatec) [inline account_cash_discount_reconcile_write_off/README.rst]: typo : an invoice

File: account_cash_discount_reconcile_write_off/README.rst [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+#. Adds a invoice that has a cash discount
```
Comment (nayatec) [inline account_cash_discount_reconcile_write_off/README.rst]: typo : Add an invoice

**Review by nayatec — CHANGES_REQUESTED** [review]
Pls fix (see comments)

**Review by hparfr — APPROVED** [review]
A part from the typo in the readme; it's good to me

Thanks @FranzPoize !

**Review by nayatec — APPROVED** [review]
LGTM thks !

**Comment by FranzPoize** [comment]
Hi @pedrobaeza could you merge this please ?

**Comment by pedrobaeza** [comment]
Merging due to the existing reviews:

/ocabot merge nobump

**Pull request merged**